docs/data/base/components/portal/portal.md

```diff
@@ -8,18 +8,28 @@ packageName: '@mui/base'
 
 # Portal
 
-<p class="description">The portal component renders its children into a new "subtree" outside of the current DOM hierarchy.</p>
+<p class="description">The <code>Portal</code> component lets you render its children into a DOM node that exists outside of its own DOM hierarchy.</p>
 
-The children of the portal component will be appended to the `container` specified.
-The component is used internally by the [`Modal`](/material-ui/react-modal/) and [`Popper`](/material-ui/react-popper/) components.
+`Portal` is a utility component built around [React's `createPortal()` API](https://reactjs.org/docs/portals.html).
+It gives you the functionality of `createPortal()` in a convenient component form.
 
-{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+:::info
+💡 According to [the React docs](https://reactjs.org/docs/portals.html), portals are useful when "you need the child element to visually 'break out' of its container."
+:::
 
-## Example
+The `Portal` component is used internally by Material UI's [`Modal`](material-ui/react-modal/) and [`Popper`](material-ui/react-popper/) components.
 
-{{"demo": "SimplePortal.js"}}
+## Basic usage
+
+Normally, children of a component are rendered within that component's DOM tree.
+But sometimes it's necessary to mount a child to a different location in the DOM.
+
+The `Portal` component accepts a `container` prop that passes a `ref` to the DOM node where its children will be mounted.
 
-## Server-side
```

Comment:
I think that we could keep a header for this and explain in more detail what we have in https://mui.com/material-ui/react-portal/#server-side. It makes it easier to skim the content

Maybe something like this?

<img width="770" alt="Screenshot 2022-05-04 at 17 35 01" src="https://user-images.githubusercontent.com/3165635/166717548-1430c37d-f681-48f6-93fd-9e6d5376d86f.png">